ps_viewedproduct.php

```diff
@@ -37,12 +37,14 @@
 class Ps_Viewedproduct extends Module implements WidgetInterface
 {
     private $templateFile;
+    private $currentProductId;
 
     public function __construct()
     {
         $this->name = 'ps_viewedproduct';
         $this->author = 'PrestaShop';
-        $this->version = '1.0.1';
+        $this->version = '1.0.2';
```

Comment:
As you modify the features of the module, a version v1.1.0 seems more suitable.

Reply:
well it is more an incompatibility with the latest PrestaShop version, do you consider this a new feature?

Reply:
From a module point of view, you update its features. :)

Reply:
Ok I will change it!

Reply:
done!